components/dashboard/src/prebuilds/list/RunPrebuildModal.tsx

```diff
@@ -76,19 +79,31 @@ export const RunPrebuildModal: FC<Props> = ({ defaultRepositoryId: defaultConfig
         <Modal visible onClose={onClose} onSubmit={handleSubmit}>
             <ModalHeader>Run a prebuild</ModalHeader>
             <ModalBody>
-                <div className="w-112 max-w-full">
+                <div className="w-112 max-w-full flex flex-col gap-8">
```

Comment:
[nit] Do we need this `gap-8` the gap seems too large for fields? (TextInputField has margin-top or padding-top itself)

<img width="798" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/4ef4e8ef-30c9-4f08-be87-90730de6f863">

Reply:
makes sense. removed it